desktop/renderer/fonticon.css

```diff
@@ -1,143 +1,167 @@
 @font-face {
-    font-family: 'kb';
```

Comment:
These whitespaces changed were in the CSS file downloaded from icomoon -- looks like they changed their CSS generation.